Rover/GCS_Mavlink.cpp

```diff
@@ -378,6 +378,7 @@ bool GCS_MAVLINK_Rover::try_send_message(enum ap_message id)
         rover.g2.windvane.send_wind(chan);
         break;
 
+#if AC_OAPATHPLANNER_ENABLED
```

Comment:
```suggestion
#if AP_OAPATHPLANNER_ENABLED
```
... and elswhere likewise.  The fact we're using this in Rover means a new define should work like this.

`AC_AVOID` is more complicated given we *also* have `AP_Avoidance` which is a *different* sort of avoidance.

Reply:
Changed.

Reply:
Thanks